ui/component/fileRenderFloating/view.jsx

```diff
@@ -18,6 +18,10 @@ import { useHistory } from 'react-router';
 import { isURIEqual } from 'util/lbryURI';
 import AutoplayCountdown from 'component/autoplayCountdown';
```

Comment:
When switching between floating and normal player, the controls are no longer visible. Not sure how to reproduce, though.

![image](https://user-images.githubusercontent.com/64950861/146150508-f5f0cb4d-6816-4a4f-933d-d063221cf84e.png)

Reply:
Was mostly testing on mobile, thx for trying web. Will try some more things to make sure nothing broke.

Reply:
Haven't been able to repo the above yet...

Reply:
Can't repro either. Moving on...